packages/rpc-graphql/src/schema/types.ts

```diff
@@ -86,4 +86,32 @@ export const typeTypeDefs = /* GraphQL */ `
         BASE_58
         BASE_64
     }
+
+    enum ExtensionType {
```

Comment:
Actually I think you'll also need to define a resolver to go to and from string values here. You'll see the others under `resolvers/types.ts`